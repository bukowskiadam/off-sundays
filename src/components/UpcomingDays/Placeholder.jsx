import React from 'react';

import style from './Placeholder.module.css';

const Placeholder = () => (
    <div className={ style.container } />
);

export default Placeholder;
