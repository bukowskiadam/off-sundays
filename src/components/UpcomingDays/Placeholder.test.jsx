import React from 'react';
import { shallow } from 'enzyme';

import Placeholder from './Placeholder';

describe('Placeholder', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Placeholder />);

        expect(wrapper).toMatchSnapshot();
    });
});
