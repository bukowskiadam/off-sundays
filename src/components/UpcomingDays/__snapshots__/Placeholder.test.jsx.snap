// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder renders correctly 1`] = `
<div
  className="container"
/>
`;
